godot-ffi/src/global.rs

```diff
@@ -283,6 +214,7 @@ mod tests {
 
     #[test]
     fn test_global_poison() {
+        std::panic::set_hook(Box::new(|_| ()));
```

Comment:
Why this change?

Reply:
Because `cargo test` was looking weird without it

Reply:
how exactly? could you clarify a bit more?

Reply:
**CAN YOU PLEASE STOP RESOLVING CONVERSATIONS WHILE QUESTIONS ARE NOT YET ANSWERED?**

Reply:
anyhow ive removed it but ill clarify a little more:
running `cargo test -- --show-output` was showing two panics which was being a little confusing

Reply:
Ok, thanks. Could you add this explanation as a comment avove `set_hook`?

Reply:
no ive removed the `set_hook` so